### ISSUE TITLE
[A11y] VA Forms System Core [Demo Content]: Repetitive heading level text and hierarchical structure issues

### ISSUE THREAD

**Issue by TiffanyPender** [department-of-veterans-affairs/vets-design-system-documentation#1112]
# Bug Report

- [X] I’ve searched for any related issues and avoided creating a duplicate issue.

*Partially related:*
* ["On this page" should be read as a heading with a level of 2 #821](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/821)
* [sitewide, design system, CMS — 508-defect-2 [AXE-CORE]: Heading levels SHOULD increase by one, On this page component department-of-veterans-affairs/va-forms-system-core#252](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/252)
* ["On this page" definition title doesn't have heading role #768](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/768)

## What happened

-[508-defect-3: ⚠️Moderate. Should be fixed in 1-3 sprints post-launch](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

As a screen reader user, the headings mapping of a page is important to understand the content of the page. Heading level text needs to be clear, concise and non-repetitive. The demo form noted below has some issues with repetitive heading level text as well as heading level hierarchical structure issues.

## Details

* ***Informational landing page*** 
    * Multiple H1 elements - there is an `H1` “Burial POC” and an `H1` “Apply for burial benefits”
    * Skips heading level hierarchy - the `H1` “Burial POC” is followed by an `H3` “Introduction Page”
    * The `H4` headings aren't really needed since the content is already nicely structured with lists. In screen reader testing, Angela found these confusing since not much content followed each `H4`.
* ***All step pages***
    * `H2` and `H3` text are almost identical
* Step 10: Review Your Application
    * Multiple `H1` elements - there is an `H1` “Burial POC” and an `H1` “Review Your Application”
    * Multiple heading elements are using the text “Review Your Application”

## What I expected to happen

- [ ] One unique `H1` per page
- [ ] All heading level text is unique
- [ ] Appropriate heading level hierarchy structure 

## Reproducing
*Environment:*

* macOS Monterey, `<Version 12.4>`
* Chrome, `<Version 102.0.5005.61>`
* VoiceOver
* Staging

*Steps to reproduce:*

1. Visit the [Forms Library Core demo form](https://staging.va.gov/burial-poc-v6/)
2. Start screen reading device of choice, and listen to each heading level and their text. Alternatively, you can also search for heading level elements in the browser dev tools OR use a heading mapping tool to view the heading level structure.
3. Verify that the text read: 
    a. Is not unique
    b. Is not structured in the appropriate hierarchical way 
    c. Includes multiple `H1`s

## Proposed Solution (if known)

Keeping in mind that the issues raised may just be issues related to the demo, but wanted to document these in case they are repeated throughout the design system. In cases where the heading level text is not unique, I recommend replacing it with unique text or removing the `H3` that mimics the `H2` text, if possible. Ensure that there is only one `H1` element on each page. Use the appropriate hierarchical heading level structure. Some guidance here:

* ***Informational landing page*** 
    * Only have one `H1` per page. "Apply for Burial Benefits" seems best as the `H1`. The "Burial POC" heading doesn't seem needed.
    * This is a demo form, so I realize the copy probably isn't as critical, but in the case of the text "Introduction Page" being a `H3`, I'd make that bold if it's needed from a templating standpoint.
    * Remove the `H4` elements and keep the text visually bold - they seem unnecessary, as the content is already nicely structured.
* ***All step pages***
    * The steps appear as an `H2` heading. For example, "Step 2 of 8: Deceased Veteran Information”. Conceptually, these are redundant because they are usually immediately followed by an `H3`. Generally, you want regular text or other content to follow a heading, not another heading. The recommendation would be to make the `H2` regular text and use other design elements to call it out visually, and make the `H3` elements that follow it `H2' elements instead.
* Step 10: Review Your Application
    * Only include one `H1` per page.
    * Remove repetitive heading elements and text
    * Also recommend making the text “On this page” an `H2` element (referenced in [issue 550 - [A11y]: VA Forms System Core [Demo Content]: Semantic HTML - Definition list used in place of more appropriate unordered list](https://github.com/department-of-veterans-affairs/va.gov-team/issues/45812))
      * *Example:*
```html
<nav>
<h2>On this page</h2>
<ul>
<li><a href=”...”>Claimant Information</a></li>
<li><a href=”...”>Deceased Veteran Information</a></li>
…
</ul>
</nav>
```

## WCAG or Vendor Guidance (optional)

* [2.4.6 Headings and Labels](https://www.w3.org/WAI/WCAG21/quickref/#headings-and-labels)
* [W3C - headings](https://www.w3.org/WAI/tutorials/page-structure/headings/)
* [A11y Project - accessible heading structure](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/)
* [Axe best practice](https://dequeuniversity.com/rules/axe/4.4/heading-order)

**Comment by humancompanion-usds** [comment]
Not going to fix Forms System Core issues. Closing due to age as won't fix.
